modules/admin-ui/src/main/webapp/login.html

```diff
@@ -84,6 +84,10 @@
     <!-- endbower -->
     <!-- endbuild -->
 
+    <!-- build:js(src/main/webapp) scripts/vendor.js -->
+    <script src="scripts/lib/chosen.js"></script>
```

Comment:
@lkiesow & @gregorydlogan Sorry for breaking the login page... :-(

I've removed that because the included file does not actually exist. Do you know why this is required?

Reply:
` <!-- build:js(src/main/webapp) scripts/vendor.js -->` should include the file. I assumed that we need chosen since the error was chosen related but maybe we could actually remove the second line.

Reply:
@lkiesow As the file scripts/lib/chosen.js simply does not exist, I think what happens is the vendor.js is generated for index.html page and does contain all libraries that page requires. The weird dependency then is that the login page depends on vendor.js generated for index.html rather than including its own dependencies. 

Anyway, it does work ;-)